x/evm/types/query.pb.go

```diff
@@ -9,10 +9,10 @@ import (
 	github_com_cosmos_cosmos_sdk_types "github.com/cosmos/cosmos-sdk/types"
 	query "github.com/cosmos/cosmos-sdk/types/query"
 	_ "github.com/cosmos/gogoproto/gogoproto"
-	grpc1 "github.com/gogo/protobuf/grpc"
-	proto "github.com/gogo/protobuf/proto"
+	grpc1 "github.com/cosmos/gogoproto/grpc"
+	proto "github.com/cosmos/gogoproto/proto"
+	github_com_cosmos_gogoproto_types "github.com/cosmos/gogoproto/types"
 	_ "github.com/gogo/protobuf/types"
```

Comment:
@spoo-bar , about _gogo proto update_

My bad, patched some in [[3]](https://github.com/dymensionxyz/ethermint/pull/27/commits/2da5b9c9e59230a8d7f25c43d5eaa6a71d47e384).
There is one place I failed to replace is this one, I have no idea why it still appears even tho I re-gen proto multiple times.

```suggestion
```

Reply:
nvm, it's gone now, by [[4]](https://github.com/dymensionxyz/ethermint/pull/27/commits/2f581674fa376e9e8ffaf7cd10404247ebcfd140)